SLHCUpgradeSimulations/Configuration/python/phase2TkFlat.py

```diff
@@ -78,127 +79,128 @@ def customise_Reco(process,pileup):
+      process.reconstruction_fromRECO.remove(process.tripletElectronSeedLayers)
+      process.reconstruction_fromRECO.remove(process.tripletElectronSeeds)
+  
+      process.reconstruction_fromRECO.remove(process.detachedQuadStep)
+      process.reconstruction_fromRECO.remove(process.detachedQuadStepClusters)
+      process.reconstruction_fromRECO.remove(process.detachedQuadStepSeedLayers)
+      process.reconstruction_fromRECO.remove(process.detachedQuadStepSeeds)
+      process.reconstruction_fromRECO.remove(process.detachedQuadStepTrackCandidates)
+      process.reconstruction_fromRECO.remove(process.detachedQuadStepTracks)
+      process.reconstruction_fromRECO.remove(process.detachedQuadStepSelector)
+  
+      process.reconstruction_fromRECO.remove(process.highPtTripletStepClusters)
+      process.reconstruction_fromRECO.remove(process.highPtTripletStepSeedLayers)
+      process.reconstruction_fromRECO.remove(process.highPtTripletStepSeeds)
+      process.reconstruction_fromRECO.remove(process.highPtTripletStepTrackCandidates)
+      process.reconstruction_fromRECO.remove(process.highPtTripletStepTracks)
+      process.reconstruction_fromRECO.remove(process.highPtTripletStepSelector)
+  
+      process.reconstruction_fromRECO.remove(process.lowPtQuadStepClusters)
+      process.reconstruction_fromRECO.remove(process.lowPtQuadStepSeedLayers)
+      process.reconstruction_fromRECO.remove(process.lowPtQuadStepSeeds)
+      process.reconstruction_fromRECO.remove(process.lowPtQuadStepTrackCandidates)
+      process.reconstruction_fromRECO.remove(process.lowPtQuadStepTracks)
+  
+      del process.iterTracking
+      del process.iterTrackingEarly
+      del process.ckftracks
+      del process.ckftracks_woBH
+      del process.ckftracks_wodEdX
+      del process.ckftracks_plus_pixelless
+      del process.trackingGlobalReco
+      del process.electronSeedsSeq
+      del process.InitialStep
+      del process.HighPtTripletStep
+      del process.LowPtQuadStep
+      del process.LowPtTripletStep
+      del process.DetachedQuadStep
+      del process.MixedTripletStep
+      del process.PixelPairStep
+      del process.TobTecStep
+      del process.earlyGeneralTracks
+      #del process.earlyMuons
+      del process.muonSeededStep
+      del process.muonSeededStepCore
+      del process.muonSeededStepCoreInOut
+      del process.muonSeededStepExtra 
+      del process.muonSeededStepDebug
+      del process.muonSeededStepDebugInOut
+      del process.ConvStep
+      
+      # add the correct tracking back in
+      process.load("RecoTracker.Configuration.RecoTrackerPhase2Tracker_cff")
+  
+      process.globalreco_tracking.insert(itIndex,process.trackingGlobalReco)
+      process.globalreco.insert(grIndex,process.globalreco_tracking)
+      #Note process.reconstruction_fromRECO is broken
+      
+      # End of new tracking configuration which can be removed if new Reconstruction is used.
```

Comment:
For phase2 I'd actually just remove all era-migrated stuff, as nobody should be using the customize functions without `trackingPhase2PU140` era. (I admit I was a bit over-precautious with phase1).